content/docs/04.5-devx/05-app-profile/02-container-deployment.md

```diff
@@ -58,6 +71,20 @@ To add a container tier to Palette Dev Engine App Profile,
        argument: HOSTNAME
  
  
-4. **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+6. Click **Review** the information and complete the app profile creation wizard. Deploy more tiers to the profile per user requirements, and Palette Clusters can be deployed using the deployed app profile. 
+
+
+## Validation
```

Comment:
Resma, the validation steps are not correct. The user should visit the Apps page from the Main Menu and check its status there.